app/routes.py

```diff
@@ -1,6 +1,15 @@
 from app import app
-from app.controllers import HomeController
+from app.controllers import HomeController, LoginController, LoginPageController
+
+from flask import request
 
 @app.route('/')
 def home():
     return HomeController().handle()
+
+@app.route('/login', methods=['GET', 'POST'])
+def login():
```

Comment:
You can have two functions for the two methods, which seems a bit cleaner than having the if statement

e.g.
```
@app.route('/login', methods=['GET'])
def login_page():

@app.route('/login', methods=['POST'])
def login():
```

Reply:
Done